packages/bruno-app/src/components/ResponsePane/ResponseClear/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { IconEraser } from '@tabler/icons';
+import { useDispatch } from 'react-redux';
+import StyledWrapper from './StyledWrapper';
+import { responseReceived } from 'providers/ReduxStore/slices/collections/index';
+
+const ResponseClear = ({ collection, item }) => {
+  const dispatch = useDispatch();
+  const response = item.response || {};
+
+  const clearResponse = () =>
+    dispatch(
+      responseReceived({
+        itemUid: item.uid,
+        collectionUid: collection.uid,
+        response: null
+      })
+    );
+
+  return (
+    <StyledWrapper className="ml-2 flex items-center">
+      <button onClick={clearResponse} disabled={!response.dataBuffer} title="Clear response">
```

Comment:
Is there any case when this button is and should be disabled? Button is displayed if any response is returned and I should be able to clear response in any such case. So I would omit setting this `disabled` parameter.

Reply:
And then you can remove also line 9

Reply:
Done!